hoomd/md/pytest/test_lj_equation_of_state.py

```diff
@@ -20,7 +20,7 @@
 @pytest.mark.parametrize(
```

Comment:
You can remove `test_lj_equation_of_state.py`. Once this branch is functional, I will test it with the longer and more complete validation tests in https://github.com/glotzerlab/hoomd-validation/.